README.md

```diff
@@ -44,100 +25,51 @@ Example:
+namely `OnDateChangedListener` is now `OnDateSelectedListener`. There are also a bunch of new apis
+for multiple selection.
 
-| Method             | Description                                                                 |
-|:-------------------|:----------------------------------------------------------------------------|
-| setMinimumDate()   | Set the earliest visible date on the calendar                               |
-| setMaximumDate()   | Set the latest visible date on the calendar                                 |
-| setSelectedDate()  | Set the date to show as selected. Must be within minimum and maximum dates. |
-| setTopbarVisible() | Set the top bar (arrows and title) as visible or gone.                      |
-| setSelectionMode() | Set the mode of selection. Choices are single, multiple or none.            |
+Also, `showOtherDates` is now a set of flags for finer control over which extra dates are shown.
 
-### Date Selection
+Major Change in 0.8.0
+---------------------
 
-We support three modes of selection: single, multiple, or none. The default is single selection.
-The mode can be changed by calling `setSelectionMode()` and passing the approprate constant (`SELECTION_MODE_NONE`, `SELECTION_MODE_SINGLE`, or `SELECTION_MODE_MULTIPLE`).
-If you change to single selection, all selected days except the last selected will be cleared.
-If you change to none, all selected days will be cleared.
+The view now responds better to layout parameters.
+The functionality is similar to how `adjustViewBounds` works with ImageView,
+where the view will try and take up as much space as necessary,
+but we base it on tile size instead of an aspect ratio.
+The exception being that if a `tileSize` is set,
+that will override everything and set the view to that size.
 
-You can set an `OnDateSelectedListener` to listen for selections, make sure to take into account multiple calls for the same date and state.
-You can manually select or deselect dates by calling `setDateSelected()`.
-There is also `setSelectedDate()` will will clear the current selection(s) and select the provided date.
+Customization
+-------------
 
-There are also: `clearSelection()`, `getSelectedDates()`, and `getSelectionMode()`; which should be self-explanitory.
+One of the aims of this library is to be customizable. The many options include:
 
+* [Custom labels for the header, weekdays, or individual days](docs/CUSTOMIZATION.md#custom-labels)
+* [Single or Multiple date selection, or disabling selection entirely](docs/CUSTOMIZATION.md)
+* [Define the view's width and height in terms of tile size](docs/CUSTOMIZATION.md#tile-size)
+* [Showing dates from other months or those out of range](docs/CUSTOMIZATION.md#showing-other-dates)
+* [Setting the first day of the week](docs/CUSTOMIZATION.md#first-day-of-the-week)
+* [Show only a range of dates](docs/CUSTOMIZATION.md#date-ranges)
+* [Customize the top bar](docs/CUSTOMIZATION.md#topbar-options)
```

Comment:
Can we order these links in the same order as the sections appear in CUSTOMIZATION.md?